README.md

```diff
@@ -83,7 +83,9 @@ Deploying to Fly.io is another great option. It's not quite one-click like Rende
 1. Click Fork > Create New Fork at the top of this repository. Pull your forked repository down to your computer (the usual git clone ...).
 1. Install the Fly command-line tool on Mac with `brew install flyctl` otherwise `curl -L https://fly.io/install.sh | sh` ([view instructions](https://fly.io/docs/hands-on/install-flyctl/))
 1. Think of an internal Fly name for your app, it has to be unique to all of Fly, and then in the root directory of the repository you pulled down, run `fly launch --build-only --copy-config --name=APP_NAME_YOU_CHOSE`
-  * Say "Yes" when it asks if you want to tweak these settings
+
+- Say "Yes" when it asks if you want to tweak these settings
```

Comment:
I reviewed the markdown and actually, the old one was incorrect, but the change is incorrect too. :) We need an extra level of indentation so that it actually nests beneath the number and doesn't restart numbering after this.

```suggestion
    - Say "Yes" when it asks if you want to tweak these settings
```